cranelift/codegen/Cargo.toml

```diff
@@ -26,6 +26,9 @@ thiserror = "1.0.4"
 byteorder = { version = "1.3.2", default-features = false }
 peepmatic-runtime = { path = "../peepmatic/crates/runtime", optional = true }
 regalloc = "0.0.24"
+rspirv = "0.6.0"
+spirv_headers = "1.4.2"
```

Comment:
Please make these optional and depend on them from the `spriv` feature.

Reply:
Done.